pkg/kf/commands/apps/proxy.go

```diff
@@ -84,26 +81,18 @@ func NewProxyCommand(p *config.KfParams, appsClient apps.Client, ingressLister i
 			}
 
 			appHost := url.Host
-
 			w := cmd.OutOrStdout()
-			fmt.Fprintf(w, "Forwarding requests from %s to %s with host %s\n", listener.Addr(), gateway, appHost)
-			fmt.Fprintln(w, "Example GET:")
-			fmt.Fprintf(w, "  curl -H \"Host: %s\" http://%s\n", appHost, gateway)
-			fmt.Fprintln(w, "Example POST:")
-			fmt.Fprintf(w, "  curl --request POST -H \"Host: %s\" http://%s --data \"POST data\"\n", appHost, gateway)
-			fmt.Fprintln(w, "Browser link:")
-			fmt.Fprintf(w, "  http://%s\n", listener.Addr())
-
-			fmt.Fprintln(w)
-
-			fmt.Fprintln(w, "\033[33mNOTE: the first request may take some time if the app is scaled to zero\033[0m")
 
 			if noStart {
-				fmt.Fprintln(cmd.OutOrStdout(), "exiting because no-start flag was provided")
+				fmt.Fprintln(w, "exiting because no-start flag was provided")
+				utils.PrintCurlExamples(w, listener, appHost, gateway, false)
 				return nil
 			}
 
-			return http.Serve(listener, createProxy(cmd.OutOrStdout(), app.Status.URL.Host, gateway))
+			utils.PrintCurlExamples(w, listener, appHost, gateway, true)
+			fmt.Fprintln(w, "\033[33mNOTE: the first request may take some time if the app is scaled to zero\033[0m")
```

Comment:
Can we stop propagating the color codes and instead use https://github.com/fatih/color?

Reply:
I think that is out of scope of this PR, I think we should create an issue for it.

Reply:
created and referenced the issue https://github.com/google/kf/issues/698